recipe/0001-Fix-linking-with-python-in-macOS.patch

```diff
@@ -0,0 +1,134 @@
+From b814562bd07d1f1eaf58a301dc5b1589f66315f6 Mon Sep 17 00:00:00 2001
+From: "Edson Tadeu M. Manoel" <tadeu@esss.co>
+Date: Wed, 20 Mar 2019 16:26:29 -0300
+Subject: [PATCH] Fix linking with python in macOS
+
+See https://blog.tim-smith.us/2015/09/python-extension-modules-os-x/
+and https://github.com/conda-forge/cgal-feedstock/pull/41
+---
+ CMakeLists.txt                       | 18 +++++++++++-------
+ cmake/Modules/SWIG_CGAL_Macros.cmake | 14 ++++++++------
+ 2 files changed, 19 insertions(+), 13 deletions(-)
+
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index c28f40b..67c023b 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -12,7 +12,11 @@ if (POLICY CMP0042)
+   cmake_policy (SET CMP0042 NEW)
+ endif()
+ 
+-option ( LINK_PYTHON_LIBRARY "Link against python libraries" ON )
++if ( APPLE )
++  option ( LINK_PYTHON_LIBRARY "Link against python libraries - NOT RECOMMENDED" OFF )
++else ()
++  option ( LINK_PYTHON_LIBRARY "Link against python libraries" ON )
++endif ()
```

Comment:
@tadeu I have asked a question in the upstream project: https://github.com/CGAL/cgal-swig-bindings/pull/114#pullrequestreview-217370631.